src/hotspot/share/runtime/java.cpp

```diff
@@ -156,6 +157,12 @@ static void print_method_profiling_data() {
   }
 }
 
+void log_vm_stats(outputStream *st) {
```

Comment:
I assume this generic name is because in the future it will print a lot more VM stats?

Reply:
Yes.